@3rdweb-sdk/react/hooks/usePayments.ts

```diff
@@ -269,6 +289,13 @@ function usePaymentsApi() {
       };
     }
 
+    if (options?.isSellerApiKey) {
+      return json as {
+        data: { data: any; decrypted_key: string };
```

Comment:
any way we can get the data type object? either hardcode or better if we can get it from the generated graphql